easybuild/easyconfigs/l/libtool/libtool-2.4.7.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'libtool'
+version = '2.4.7'
+
+homepage = 'https://www.gnu.org/software/libtool'
+
+description = """
+ GNU libtool is a generic library support script. Libtool hides the complexity
+ of using shared libraries behind a consistent, portable interface.
+"""
+
+toolchain = SYSTEM
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['04e96c2404ea70c590c546eba4202a4e12722c640016c12b9b2f1ce3d481e9a8']
+
+dependencies = [
+    ('M4', '1.4.18'),
```

Comment:
@dithwick why not use the more recent version `1.4.19` here?

Reply:
Hi @smoors.

I was going to originally but then in the process of putting together the system toolchain version of the Autotools-20220317 release, I noticed Autoconf-2.71.eb uses the 1.4.18 release of M4 so figured if I made it the 1.4.18 release here, only one version of M4 needed to be built. Happy to change it to 1.4.19, or should I do an update to Autoconf-2.71.eb so it uses 1.4.19 instead?

Reply:
no it's ok, thanks.